pkg/adapter/v2/config.go

```diff
@@ -58,6 +61,11 @@ type EnvConfig struct {
 	// This is used to configure the logging config, the config is stored in
 	// a config map inside the controllers namespace and copied here.
 	LoggingConfigJson string `envconfig:"K_LOGGING_CONFIG" required:"true"`
+
+	// TracingConfigJson is a json string of tracing.Config.
+	// This is used to configure the tracing config, the config is stored in
+	// a config map inside the controllers namespace and copied here.
+	TracingConfigJson string `envconfig:"K_TRACING_CONFIG" required:"true"`
```

Comment:
It's a little bit cumbersome in my opinion to mark every env var as `required`. Many adapters depend on this lib, but only a few care about configuring tracing. I believe most would be just fine defaulting to sane defaults.